sys/log/util/cbor_reader/include/cbor_reader.h

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I think `log_cbor_reader` would be a better name for this directory than `cbor_reader`.  If I see
```
#include "cbor_reader.h"
```

It wouldn't occur to me that the file includes a log header.

Also, I think there should be an extra subdirectory under `include` to maintain consistency, i.e.,
```
 sys/log/util/cbor_reader/include/cbor_reader/...
```
```
#include "cbor_reader/cbor_reader.h"
```

Reply:
done